qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -375,328 +294,322 @@ def _unpack_num(num, sig):
             num //= k
         return res
 
-    @staticmethod
-    def _unpack_num_multi_sigs(num, sigs):
-        """Returns the result of `_unpack_num` on one of the
-        signatures in `sigs`
-        """
-        for i, sig in enumerate(sigs):
-            sig_size = 1
-            for k in sig:
-                sig_size *= k
-            if num < sig_size:
-                return [i] + CliffordUtils._unpack_num(num, sig)
-            num -= sig_size
-        return None
-
-    def num_from_clifford_single_gate(
-        self, inst: Instruction, qubits: List[int], rb_num_qubits: int
-    ) -> int:
-        """
-        This method does the reverse of clifford_1_qubit_circuit and clifford_2_qubit_circuit -
-        given a clifford, it returns the corresponding integer, with the mapping
-        defined in the above method.
-
-         Returns:
-            An integer representing a Clifford consisting of a single gate.
 
-        Raises:
-            QiskitError: if the input instruction is not a Clifford instruction.
-            QiskitError: if rz is given with a angle that is not Clifford.
-        """
-        name = inst.name
-        if name == "delay":
-            return 0
-
-        clifford_gate_set = {key[0] for key in self.CLIFF_SINGLE_GATE_MAP[rb_num_qubits]}
-        if name == "rz":
-            # The next two are identical up to a phase, which makes no difference
-            # for the associated Cliffords
-            if isclose(inst.params[0], np.pi) or isclose(inst.params[0], -np.pi):
-                map_index = "z"
-            elif isclose(inst.params[0], np.pi / 2):
-                map_index = "s"
-            elif isclose(inst.params[0], -np.pi / 2):
-                map_index = "sdg"
-            else:
-                raise QiskitError("wrong param {} for rz in clifford".format(inst.params[0]))
-        elif name in clifford_gate_set:
-            map_index = name
-            if name == "cz":
-                # for cz we save only [0, 1] since it is a symmetric operation
-                qubits = [min(qubits), max(qubits)]
+# Constant mapping from 1Q single Clifford gate to 1Q Clifford number.
+# This table must be generated using `data.generate_clifford_data.gen_cliff_single_1q_gate_map`.
+_CLIFF_SINGLE_GATE_MAP_1Q = {
+    ("id", (0,)): 0,
+    ("h", (0,)): 1,
+    ("sxdg", (0,)): 2,
+    ("s", (0,)): 4,
+    ("x", (0,)): 6,
+    ("sx", (0,)): 8,
+    ("y", (0,)): 12,
+    ("z", (0,)): 18,
+    ("sdg", (0,)): 22,
+}
+# Constant mapping from 2Q single Clifford gate to 2Q Clifford number.
```

Comment:
```suggestion
# Constant mapping from 2Q single Clifford gate to 2Q Clifford numerical identifier.
```
I think number is confusing terminology.

Reply:
Resolved in 851cf64